ee/ucp/admin/configure/deploy-route-reflectors.md

```diff
@@ -1,137 +1,133 @@
-      # Mount in the etcd TLS secrets.
-        - name: etcd-certs
-          secret:
-            secretName: calico-etcd-secrets
-  ```
-
-Then, deploy the DaemonSet using:
-
-```
-kubectl create -f calico-rr.yaml
-```
+    updateStrategy:
+        type: RollingUpdate
+    selector:
+        matchLabels:
+        k8s-app: calico-rr
+    template:
+        metadata:
+        labels:
+            k8s-app: calico-rr
+        annotations:
+            scheduler.alpha.kubernetes.io/critical-pod: ''
+     spec:
+        tolerations:
+            - key: com.docker.ucp.kubernetes.calico/route-reflector
+            value: "true"
+            effect: NoSchedule
+         hostNetwork: true
+        containers:
+            - name: calico-rr
+            image: calico/routereflector:v0.6.1
+              env:
+                - name: ETCD_ENDPOINTS
+                  valueFrom:
+                    configMapKeyRef:
+                      name: calico-config
+                      key: etcd_endpoints
+                - name: ETCD_CA_CERT_FILE
+                  valueFrom:
+                    configMapKeyRef:
+                      name: calico-config
+                      key: etcd_ca
+                # Location of the client key for etcd.
+                - name: ETCD_KEY_FILE
+                  valueFrom:
+                    configMapKeyRef:
+                      name: calico-config
+                      key: etcd_key # Location of the client certificate for etcd.
+                - name: ETCD_CERT_FILE
+                  valueFrom:
+                    configMapKeyRef:
+                      name: calico-config
+                      key: etcd_cert
+                - name: IP
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: status.podIP
+              volumeMounts:
+                - mountPath: /calico-secrets
+                  name: etcd-certs
+              securityContext:
+                privileged: true
+          nodeSelector:
+            com.docker.ucp.kubernetes.calico/route-reflector: "true"
+          volumes:
+          # Mount in the etcd TLS secrets.
+            - name: etcd-certs
+              secret:
+                secretName: calico-etcd-secrets
+      ```
+
+2. Deploy the DaemonSet by running:
+
+    ```
+    kubectl create -f calico-rr.yaml
+    ```
 
 ## Configure calicoctl
 
-To reconfigure Calico to use Route Reflectors instead of a node-to-node mesh,
-you'll need to tell `calicoctl` where to find the etcd key-value store managed
-by UCP.  From a CLI with a UCP client bundle, create a shell alias to start
-`calicoctl` using the `{{ page.ucp_org }}/ucp-dsinfo` image:
+To reconfigure Calico to use route reflectors instead of a node-to-node mesh,
+you need to tell `calicoctl` where to find the etcd key-value store managed
+by UCP. From a CLI configured with a UCP client bundle, create a shell alias to start `calicoctl` using the `{{ page.ucp_org }}/ucp-dsinfo` image:
```

Comment:
Edit the sentence to read: "To configure Calico to use route reflectors instead of a node-to-node mesh, `calicoctl` needs to be able to locate the etcd key-value store managed by UCP."